contrib/templates/experimental/scala-job/template/build.sbt.tmpl

```diff
@@ -0,0 +1,22 @@
+scalaVersion := "{{template `scala_version` .}}"
```

Comment:
Could you add a comment to the top of this file, explaining what it's for? See https://github.com/databricks/cli/blob/main/libs/template/templates/default-python/template/%7B%7B.project_name%7D%7D/setup.py.tmpl#L2 for inspiration

(and yes, sbt users will know what this is, but it's a helpful general convention to offer some guidance for all configuration files)